platform/windows/detect.py

```diff
@@ -169,7 +169,7 @@ def get_flags():
 
 	return [
 		('freetype','builtin'), #use builtin freetype
-		('openssl','builtin'), #use builtin openssl
+		#('openssl','builtin'), #use builtin openssl
```

Comment:
If those local changes are important, do ``git stash``. If not, just revert them with ``git checkout -- .``.